src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -6,12 +6,15 @@
 		<template #content>
 			<kv-page-container>
 				<kv-grid
-					class="tw-grid-cols-12 tw-gap-2 tw-items-center"
-					:class="{ 'tw-grid-cols-1 tw-mx-auto': singleColumn }"
+					class="tw-grid-cols-12 tw-gap-2 tw-items-center tw-justify-items-center"
 					:style="customGridStyles"
 				>
 					<div
-						class="tw-mx-auto tw-col-span-12 md:tw-col-span-6"
+						class="tw-mx-auto tw-col-span-12"
+						:class="{
+							'md:tw-col-span-6': !singleColumn
+						}"
+						:style="maxWidthStyles"
```

Comment:
It's kind of odd that the maxWidthStyles are not applying to the entire content area...I think there are designs in the pipeline that may need to restrict the width in that way.

Reply:
Good feedback, will work on that aspect